content/docs/application-mobility/use_cases.md

```diff
@@ -0,0 +1,73 @@
+
+1. If Velero is not installed in the default `velero` namespace, set this environment variable to the namespace where it is installed:
+    ```
+    export VELERO_NAMESPACE=application-mobility 
+    ```
+1. On the source cluster, create a Backup by providing a name and the included namespace where the application is installed. The application and its data will be available in the object store bucket and can be restored at a later time.
+    ```
+    dellctl backup create backup1 --include-namespaces demo1 --namespace application-mobility
+    ```
+1. Monitor the backup status until it is marked as Completed.
+    ```
+    dellctl backup get --namespace application-mobility
+    ```
+1. If the Storage Class name on the target cluster is different than the Storage Class name on the source cluster where the backup was created, a mapping between source and target Storage Class names must be defined. See [Changing PV/PVC Storage Classes](#changing-pvpvc-storage-classes).
+1. The application and its data can be restored on either the same cluster or another cluster by referring to the backup name and providing an optional mapping of the original namespace to the target namespace.
+    ```
+    dellctl restore create restore1 --from-backup backup1 \
+        --namespace-mappings "demo1:restorens1" --namespace application-mobility
+    ```
+1. Monitor the restore status until it is marked as Completed.
+    ```
+    dellctl restore get --namespace application-mobility
+    ```
+
+## Clone an Application
+This example details the steps when an application in namespace `demo1` is cloned from a source cluster to a target cluster in a single operation. In this sample, both Application Mobility and Velero are installed in the `application-mobility` namespace.
+
+1. If Velero is not installed in the default `velero` namespace, set this environment variable to the namespace where it is installed:
+    ```
+    export VELERO_NAMESPACE=application-mobility 
+    ```
+1. Register both the source and target clusters
+    ```
+    dellctl cluster add -n cluster1 -f ~/kubeconfigs/cluster1-kubeconfig
+    dellctl cluster add -n cluster2 -f ~/kubeconfigs/cluster2-kubeconfig
+    ```
+1. If the Storage Class name on the target cluster is different than the Storage Class name on the source cluster where the backup was created, a mapping between source and target Storage Class names must be defined. See [Changing PV/PVC Storage Classes](#changing-pvpvc-storage-classes).
+1. Create a Backup by providing a name, the included namespace where the application is installed, the source cluster, and the target cluster and namespace mapping where the application will be restored.
+    ```
+    dellctl backup create backup1 --include-namespaces demo1 --cluster-id cluster1 \
```

Comment:
JFYI: if the the source cluster is on the same host where dellctl is run, then it is really not required to register the source cluster and also not required to use --cluster-id flag(by default the command is run against local cluster). Update the below "dellctl backup get" command accordingly.

Reply:
Removed references to registering source cluster and updated the "dellctl backup get" command to monitor for the restore on the target cluster.